src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String colour;
+
+    public void setColour(String colour) {
+        this.colour = colour;
+    }
+
+    public String getColour() {
+        return colour;
+    }
+
+    public abstract void draw();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes)